client/src/components/Ticketing/donation/OnlyDonatePage.tsx

```diff
@@ -46,35 +63,203 @@ export default function OnlyDonationPage(): ReactElement {
     }
   };
 
+  const handleRadioChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setDonationPeriod(event.target.value);
+    setAmount(0);
+    console.log(amount);
+  };
+
+  const handleDonationButtonClick = (amount: number) => {
+    setAmount(amount);
+  };
+
   return (
-    <div className='w-full py-[5rem] px-[1rem] tab:px-[5rem] flex flex-col items-center'>
+    <div className='w-full max-w-6xl mx-auto py-[5rem] px-[1rem] tab:px-[5rem] flex flex-col items-center'>
+      {/* Back button*/}
       <div className='w-full flex flex-row mb-8'>
-        <button onClick={() => history('/')} className='bg-blue-500 mt-10 hover:bg-blue-600 px-3 py-2 rounded-xl flex flex-row items-center text-zinc-100'>
-          <svg xmlns="http://www.w3.org/2000/svg" className="h-5 w-5" viewBox="0 0 20 20" fill="currentColor">
-            <path fillRule="evenodd" d="M9.707 14.707a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 1.414L7.414 9H15a1 1 0 110 2H7.414l2.293 2.293a1 1 0 010 1.414z" clipRule="evenodd" />
+        <button
+          onClick={() => history('/')}
+          className='bg-blue-500 mt-10 hover:bg-blue-600 px-3 py-2 rounded-xl flex flex-row items-center text-white'
+        >
+          <svg
+            xmlns='http://www.w3.org/2000/svg'
+            className='h-5 w-5'
+            viewBox='0 0 20 20'
+            fill='currentColor'
+          >
+            <path
+              fillRule='evenodd'
+              d='M9.707 14.707a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 1.414L7.414 9H15a1 1 0 110 2H7.414l2.293 2.293a1 1 0 010 1.414z'
+              clipRule='evenodd'
+            />
           </svg>
           back to Events
         </button>
       </div>
-
-      <div className='text-3xl text-black font-bold mb-12'>Thank you for donating</div>
-      <div className='flex flex-col w-full items-start'>
-        <div className='text-sm text-zinc-600 ml-2'>Donation Amount $</div>
-        <input
-          placeholder="Donation Amount"
-          onChange={(e) => setAmount(+e.target.value)}
-          type="number"
-          className='w-full mb-7 bg-zinc-200 text-black p-5 rounded-xl'
-          value={amount || null}
-        />
+      {/* Image */}
+      <div>
+        <img src={DonationImage} alt='Donation Page Image' />
+      </div>
+      {/* Header text */}
+      <div className='text-center text-2xl'>
+        <h1>
+          Double your impact! <br />
+          Donate before June 30th and your gift will be matched 100%
+        </h1>
+      </div>
+      <DonationIntro />
+      <hr className='w-full border border-t border-zinc-300 my-4'></hr>
```

Comment:
Would it be possible to make this a void tag since the body is empty?

Reply:
I'm not the front end guy who designed this so, no. They are no longer in the capstone. You need to assign this to a new student in the capstone to work on as I only do the backend part, and I am working as a volunteer at this point as I finished this last term.

Please assign this front end tasks to a new capstone member please and thank you.

Reply:
Thanks for following up on this Justin. I'll address Ben's comment so that we can close out this PR.

Reply:
Made that a void tag https://github.com/WonderTix/WonderTix/pull/517/commits/48dbdd942b1b06e07e18ea566bf397b636f546cd